tests/cupy_tests/statistics_tests/test_order.py

```diff
@@ -392,9 +379,9 @@ def test_ptp_all_nan(self, xp, dtype):
     'magic_value': (-29, -53, -207, -16373, -99999,)
 }))
 @testing.gpu
-class TestPercentileMonotonic(unittest.TestCase):
+class TestPercentileMonotonic:
 
-    @testing.with_requires('numpy>=1.20')
+    @testing.with_requires('numpy>=1.20', 'numpy<1.22')
```

Comment:
It could be simplified as
```suggestion
    @testing.with_requires('numpy>=1.20,<1.22')
```

https://setuptools.pypa.io/en/latest/pkg_resources.html#requirements-parsing

Reply:
Thanks!